### PR TITLE
REM-551

### PR DESCRIPTION
**Pull request by imtithalhalalo**
schedules tab data grid that have 4 columns
page: 'plants'

### CODE REVIEW

File: src/pages/plants/Forms/SchedulesTab.js [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+    }),
+    enableReinitialize: false,
+    validateOnChange: true,
+    maxAccess,
+    onSubmit: async values => {
+      await post(values.schedules)
+    }
+  })
+
+  const post = async obj => {
```
Comment (RebeccaFrem02) [inline src/pages/plants/Forms/SchedulesTab.js:55]: name this function postSchedules if you can

File: src/pages/plants/Forms/SchedulesTab.js [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+          formik.setValues({
+            ...formik.values,
+            schedules: mergedSchedules.map((items, index) => ({
+              ...items,
+              id: index + 1
+            }))
+          })
+          setStore(prevStore => ({
+            ...prevStore,
+            schedules: mergedSchedules.map((items, index) => ({
```
Comment (RebeccaFrem02) [inline src/pages/plants/Forms/SchedulesTab.js:169]: why are we putting the schedules in store are we using it anywhere else?

File: src/pages/plants/Windows/PlantWindow.js [hunk 3]
```diff
@@ -8,17 +8,19 @@ import { SystemRepository } from 'src/repositories/SystemRepository'
 import toast from 'react-hot-toast'
 import AddressForm from 'src/components/Shared/AddressForm'
 import { ControlContext } from 'src/providers/ControlContext'
+import SchedulesTab from '../Forms/SchedulesTab'
 
 const PlantWindow = ({ labels, editMode, maxAccess, recordId, height }) => {
   const [store, setStore] = useState({
     recordId: recordId || null,
```
Comment (omarZrayka23) [inline src/pages/plants/Windows/PlantWindow.js:15]: can u clear || null

**Comment by omarZrayka23** [comment]
cell field type must be  time

File: src/pages/plants/Forms/SchedulesTab.js [hunk 4]
```diff
@@ -0,0 +1,204 @@
...
+    enableReinitialize: false,
+    validateOnChange: true,
+    maxAccess,
+    onSubmit: async values => {
+      await postSchedules(values.schedules)
+    }
+  })
+
+  const postSchedules = async obj => {
+    const data = {
```
Comment (omarZrayka23) [inline src/pages/plants/Forms/SchedulesTab.js:56]: u can put this in submit direct without this function

**Review by omarZrayka23 — APPROVED** [review]

**Pull request merged**